SCAutolib/controller.py

```diff
@@ -82,7 +86,7 @@ def prepare(self, force: bool, gdm: bool, install_missing: bool):
             u = self.setup_user(usr, force=force)
             self.enroll_card(u)
 
-    def setup_system(self, install_missing: bool, gdm: bool):
+    def setup_system(self, install_missing: bool, gdm: bool, graphical: bool):
```

Comment:
Is `--gdm` actually used in any tests? It seems somewhat redundant with the `--graphical` option.